peer/network.go

```diff
@@ -428,13 +424,6 @@ func (n *network) SetRequestHandler(handler message.RequestHandler) {
 	n.requestHandler = handler
 }
 
-func (n *network) SetCrossChainRequestHandler(handler message.CrossChainRequestHandler) {
-	n.lock.Lock()
-	defer n.lock.Unlock()
-
-	n.crossChainRequestHandler = handler
```

Comment:
could we also remove `crossChainRequestHandler` from the struct?

Reply:
thanks